### PR TITLE
Add 4.16 nightly run for winc vsphere

### PR DESCRIPTION
**Pull request by weinliu**

### CODE REVIEW

**Comment by weinliu** [comment]
@rrasouli @liangxia could you help take a look? Thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml [hunk 1]
```diff
@@ -3448,6 +3448,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet
+- as: vsphere-ipi-ovn-winc-f7
+  cron: 35 3 5 * *
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:3452]: Test name show as `f7` (weekyly), while cron entry is monthly

Comment (weinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml]: Thanks, updated to `3 18 7,16,23,30 * *`

**Pull request merged**